File: spec/code_samples/Node_unirest/list/post.js
var unirest = require("unirest");

var req = unirest("POST", "https://analyzemail.com/api/v1/list");

req.query({
  "username": "SOME_STRING_VALUE"
});

req.headers({
  "content-type": "application/json"
});

req.type("json");
req.send({
  "filename": "<ADD STRING VALUE>",
  "emails": [
    "<ADD STRING VALUE>"
  ]
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Python_python3/list/post.py
import http.client

conn = http.client.HTTPSConnection("analyzemail.com")

payload = "{\"filename\":\"<ADD STRING VALUE>\",\"emails\":[\"<ADD STRING VALUE>\"]}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/api/v1/list?username=SOME_STRING_VALUE", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))